.browserslistrc

```diff
@@ -0,0 +1,5 @@
+Chrome >= 60
```

Comment:
These will need to be maintained manually now, since they're not relative like before (last two versions). Is the relative syntax not supported in the new version?

Reply:
Hmm I might have been playing around with something other than just the Webpack upgrade here... I might have been trying to play around with https://developers.google.com/web/fundamentals/primers/modules as well